src/events/PollStartEvent.ts

```diff
@@ -173,11 +173,11 @@ export class PollStartEvent extends ExtensibleEvent<M_POLL_START_EVENT_CONTENT>
 
     /**
      * Creates a new PollStartEvent from question, answers, and metadata.
-     * @param {string} question The question to ask.
-     * @param {string} answers The answers. Should be unique within each other.
-     * @param {KNOWN_POLL_KIND|string} kind The kind of poll.
-     * @param {number} maxSelections The maximum number of selections. Must be 1 or higher.
-     * @returns {PollStartEvent} The representative poll start event.
+     * @param question The question to ask.
+     * @param answers The answers. Should be unique within each other.
+     * @param kind The kind of poll.
+     * @param maxSelections The maximum number of selections. Must be 1 or higher.
+     * @returns The representative poll start event.
```

Comment:
oh, hmm, sorry - I just realized this change is in here. Ideally we keep the jsdoc types until the project settles on a documentation renderer.

It's redundant with typescript, but not all documentation renderers support TS types.